index.js

```diff
@@ -1,13 +1,28 @@
 const core = require("@actions/core")
+const github = require('@actions/github')
+const { Octokit } = require("@octokit/rest")
 const { generate } = require("./lib/generate")
 
 async function run() {
   try {
-    const commits = JSON.parse(core.getInput("commits"))
+    const octokit = new Octokit();
+
     const previousReleaseTagNameOrSha = core.getInput("previousReleaseTagNameOrSha")
     const nextReleaseTagName = core.getInput("nextReleaseTagName")
     const nextReleaseName = core.getInput("nextReleaseName")
-    const configPath = core.getInput("configPath")
+    const configPath = core.getInput("configFilePath")
```

Comment:
```suggestion
    const configPath = core.getInput("configPath")
```

Reply:
The variable name in `action.yml` is `configFilePath` I changed this to match that. Do you want me to change both to `configPath`?

Reply:
No, I had it backwards 🤦.Thanks for catching it!